lib/ansible/modules/cloud/google/gcp_compute_instance.py

```diff
@@ -577,6 +577,11 @@
     - auto_delete: 'true'
       boot: 'true'
       source: "{{ disk }}"
+    - auto_delete: 'true'
```

Comment:
`gcp_compute_instance` is auto generated module. Please edit this information in https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/examples/ansible/instance.yaml